crowdin.yml

```diff
@@ -0,0 +1,46 @@
+#
+# Your Crowdin credentials
+#
+"base_path": "."
+"base_url": "https://api.crowdin.com"
+"commit_message": 'New translations: %original_file_name% (%language%)'
+"append_commit_message": false
+
+#
+# Choose file structure in Crowdin
+# e.g. true or false
+#
+"preserve_hierarchy": true
+
+#
+# Files configuration
+#
+files: [
+ {
+  #
+  # Source files filter
+  # e.g. "/resources/en/*.json"
+  #
+  "source" : "/translation/source/*.json",
+
+  #
+  # Where translations will be placed
+  # e.g. "/resources/%two_letters_code%/%original_file_name%"
+  #
+  "translation" : "/translation/dest/%locale%/%file_name%.%file_extension%",
+
+  "labels" : ["mobile"],
+
+  #
+  # File type
+  # e.g. "json"
+  #
+  "type" : "json",
```

Comment:
At the time we chose xml for lila, because it was the simplest format with support for plural forms. No plurals needed in lichobile, or does json support plurals, now?

Reply:
Good question. We've had to [flatten plurals](https://github.com/veloce/lichobile/blob/ae09c60192bffdbe1161af5c430afedacf2893dd/scripts/i18n-update.ts#L118-L122) from the lila XML anyway, so same difference on the lichobile side. It'll just end up being [multiple keys](https://github.com/veloce/lichobile/blob/ae09c60192bffdbe1161af5c430afedacf2893dd/src/i18n.ts#L204).

Looks like Crowdin consolidates editing plurals grouped under the same key, so if that's desirable over totally separate keys, I can switch this to XML

Reply:
In fact... we could even switch to [yaml](https://support.crowdin.com/file-formats/yaml/), which I find significantly more concise than XML...

Reply:
Looks like yaml would work. too. The reason it can't be flattened in the source, only the translation, is that some planguages have more plural forms than others. Example:

https://github.com/ornicar/lila/blob/ca3970720ef808800b7b3b72334da345d38fe8bd/translation/dest/site/ru-RU.xml#L37-L42

So `pluralKey` (https://github.com/veloce/lichobile/blob/b76cccb0d6f7faa478ee1072c8abdbf4c850299e/src/i18n.ts#L35) can be more than just `...:one` and `...:other`, depending on the language.

Reply:
#1618 up to switch to YAML